upgrade/sql/1.7.7.0.sql

```diff
@@ -297,7 +296,6 @@ ALTER TABLE `PREFIX_timezone` CHANGE `name` `name` varchar(32) CHARACTER SET utf
 ALTER TABLE `PREFIX_attribute_group` CHANGE `group_type` `group_type` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL;
 ALTER TABLE `PREFIX_search_word` CHANGE `word` `word` varchar(30) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL;
 ALTER TABLE `PREFIX_meta` CHANGE `page` `page` varchar(64) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL;
-ALTER TABLE `PREFIX_statssearch` CHANGE `keywords` `keywords` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL;
```

Comment:
We could add a PHP function for this something like:
```php
execute_sql_if_table_exists('statssearch', 'ALTER TABLE `PREFIX_statssearch` CHANGE `keywords` `keywords` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL;');
```

With a function that would look like this:
```php
function execute_sql_if_table_exists($tableToCheck, $sqlQuery): void
{
    $prefixedTable = _DB_PREFIX_ . $tableToCheck;
    // Check if table exists, if it doesn't return
    // If not execute the $sqlQuery (don't forget to replace placeholders like PREFIX
}
```

The `PrestaShopBundle\Install\SlqLoader` should be used in this version as it handles the replacement of placeholders and avoid multiple queries at the same time